.github/workflows/visual-diff.yml

```diff
@@ -12,8 +12,8 @@ on:
 
 jobs:
   visual-diff:
-    # This is the opposite condition as acceptance.yml/visual-diff
-    if: github.head_repository.full_name != 'getsentry/sentry'
+    # https://docs.github.com/en/enterprise-server@3.2/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow
+    if: github.repository == 'getsentry/sentry' && github.event.workflow_run.conclusion == 'success'
```

Comment:
I'm going back to the original condition `github.repository == 'getsentry/sentry'`.

This is new: `github.event.workflow_run.conclusion == 'success'`